xml/Microsoft.Vsa/VsaModule.xml

```diff
@@ -47,7 +47,7 @@
         <Parameter Name="bIsVsaModule" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="bIsVsaModule">To be added.</param>
+        <param name="bIsVsaModule">True if the module is a VSA module, false otherwise</param>
         <summary>This member supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
```

Comment:
Check how we write summaries
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Constructor